src/Collapse.tsx

```diff
@@ -206,23 +226,25 @@ class Collapse extends React.Component {
         onExit={handleExit}
         onExiting={handleExiting}
       >
-        {(state, innerProps) =>
-          React.cloneElement(children, {
+        {(state, innerProps) => {
+          return React.cloneElement(children as any, {
```

Comment:
can children be typed as `React.ReactElement` to avoid this?

Reply:
Not without refactoring Collapse into an FC. ClassComponents' `props` are typed like
```
readonly props: Readonly<P> & Readonly<{ children?: ReactNode }>;
```